src/unstract/sdk/index.py

```diff
@@ -186,123 +187,138 @@ def index_file(
             )
             raise SdkError(f"Error loading {vector_db_instance_id}: {e}")
 
-        # Checking if document is already indexed against doc_id
-        doc_id_eq_filter = MetadataFilter.from_dict(
-            {"key": "doc_id", "operator": FilterOperator.EQ, "value": doc_id}
-        )
-        filters = MetadataFilters(filters=[doc_id_eq_filter])
-        q = VectorStoreQuery(
-            query_embedding=embedding.get_query_embedding(" "),
-            doc_ids=[doc_id],
-            filters=filters,
-        )
-
-        doc_id_found = False
         try:
-            n: VectorStoreQueryResult = vector_db.query(query=q)
-            if len(n.nodes) > 0:
-                doc_id_found = True
-                self.tool.stream_log(f"Found {len(n.nodes)} nodes for {doc_id}")
-            else:
-                self.tool.stream_log(f"No nodes found for {doc_id}")
-        except Exception as e:
-            self.tool.stream_log(
-                f"Error querying {vector_db_instance_id}: {e}", level=LogLevel.ERROR
+            # Checking if document is already indexed against doc_id
```

Comment:
The changes that are showing up here is due to the added try-finally block to close the vector db connection. Mainly white space indentation. Core-logic is the same.